docs/product-user-guide/src/main/asciidoc/guided-decision-tables-create-proc.adoc

```diff
@@ -7,21 +7,22 @@ You can use guided decision tables to define rule attributes, meta-data, conditi
 . Go to *Menu* -> *Design* -> *Projects* and select your project.
 . Click *Create New Asset* -> *Guided Decision Table*.
 . Enter an informative *Guided Decision Table* name and select the appropriate *Package*.
-. Select *Use Wizard* to finish setting up the table in the Wizard, or leave this option unselected to finish creating the table and specify remaining configurations in the guided decision table editor.
-+
-The Wizard helps you define your imports, facts, patterns, and columns, but not the rows. Rows are added in the guided decision table editor after initial table setup.
```

Comment:
Why delete this sentance?

Reply:
Moved, not deleted. The doc covers the wizard a bit later, where there is a screenshot of the wizard. Covering the wizard here, next to a different screenshots, was in my view confusing, so I moved the information lower, slightly rephrasing it.